sonic-net/sonic-buildimage
# [Marvell-armhf] Update MRVL_PRESTERA_DRIVER

#### Why I did it
Update SDK driver version to MRVL_PRESTERA_DRIVER_1.7_1.
Enable SDK DMA driver with reserved memory in DTS.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

#### How to verify it
Build the sonic image with updated sdk driver and loaded on hardware to check interfaces up.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
